react-charts/stack-chart/step5.md

```diff
@@ -1,34 +1,58 @@
-Now we'll add tooltips to the `Chart` component...
+Add axis labels to the stack chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the code for the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It should look like this:
 
 <pre class="file">
 &lt;Chart
+  containerComponent={&lt;ChartVoronoiContainer labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`} /&gt;}

```

Comment:
Change this so its not on 1 line:
<img width="471" alt="Screen Shot 2019-12-09 at 9 23 32 AM" src="https://user-images.githubusercontent.com/20118816/70443129-99a57d00-1a65-11ea-879f-ded982e66029.png">

Reply:
matches chartArea exactly now